wiki/osu!tourney/osu!lazer_tournament_client/en.md

```diff
@@ -0,0 +1,86 @@
+
+## Managing Tournaments
+
+Tournament configurations for osu!lazer are stored in `%APPDATA%/osu/tournaments`. On first start up of the client a directory called `default` is made inside this folder. A user can maintain multiple tournament configurations and switch between them as needed to get the appropriate customisations applied.
+
+To create a new tournament configuration, create a new directory in the `tournaments` directory with the name of your tournament.
+
+Inside a tournament configuration the necessary assets can be supplied to display flags, videos and mod icons for the mappool. Each category of asset has its own folder: 
+
+- your-tournament
+    - flags
+    - mods
+    - videos
+
+## Customisation
+
+The osu!lazer tournament client can be customised by providing it with custom flags, mod icons, and video files. These will display in the respective scenes as needed.
+
+### Flags
+
+For custom flags, `.jpg` and `.png` files are accepted. Flag images should be at least 140x94, maintaining an aspect ratio close to this specification for best results.
+
+An example of a flag using the correct specifications is this flag of Australia:
+
+![][flag_AU]
+
+Flags have to be placed in `<your-tournament>/flags`. The flags can then be referenced in the Team Editor by their filenames without the file extension.
+
+### Mods
+
+For custom mod icons, `.jpg` and `.png` files are accepted. The resolution can be anything and the client will fit it in the beatmap panel. For reference, a beatmap panel at 1920x1080 is 563x60 pixels.
+
+Mod icons have to be placed in `<your-tournament>/mods`. The mods can then be referenced by their filenames without the file extension in the Rounds Editor and in the Seeding Results Editor.
+
+### Videos
+
+Videos can be displayed in the backgrounds of scenes visible to the stream. These will automatically be looped in the client once the end of the video is reached.
+
+Note: The client decodes video files using software decoding, so depending on the usage scenario performance may vary. 
+
+The files have to adhere to the following specifications:
+
+- 16:9 aspect ratio, for example 1280x720 or 1920x1080
+- `mp4`, `m4v`, or `avi` file extension
+- Video codec: H.264, Audio codec: none
+
+The video files have to placed in `<your-tournament>/videos` and specific names are required for the correct functionality.
+
+| Scene | File(s) |
+| :-- | :-- |
+| Schedule | `schedule` |
+| TeamIntro | `teamintro` |
+| Seeding | `seeding` |
+| MapPool | `mappool` |
+| Gameplay | `gameplay` |
+| Win | `teamwin-red`, `teamwin-blue` |
+| Drawings | `main` |
+| Showcase | `showcase` |
+
+[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
+
+## Editors
+
+The osu!lazer tournament client has three editors that can be used to provide the following information about a tournament: the teams and its players, the rounds and their accompanying mappool and the bracket. 
+
+### Team Editor
```

Comment:
Is this document supposed to end here? Should this not be added as a separate step if you don't intend on finishing it?